### PR TITLE
chore: remove default exports [P1]

### PR DESCRIPTION
**Pull request by layershifter**
#### Description of changes

This PR is a first round of `default` exports removal, see https://github.com/microsoft/fluentui/issues/12953#issuecomment-633970215.

### CODE REVIEW

File: packages/fluentui/react-bindings/src/index.ts [hunk 1]
```diff
@@ -11,25 +11,24 @@ export { default as FocusZone } from './FocusZone/FocusZone';
 export * from './FocusZone/FocusZone.types';
```
Comment (miroslavstastny) [inline packages/fluentui/react-bindings/src/index.ts:11]: There are still default exports in `react-bindings`:
image

Comment (layershifter) [reply packages/fluentui/react-bindings/src/index.ts:11]: Yep, they will be handled separately as they require changes in `getComponentInfo()`: https://github.com/microsoft/fluentui/blob/d55eecf434dfe269ecbf6f412dcde73a13dace3e/scripts/gulp/plugins/util/getComponentInfo.ts#L64

All `export default` will be catched by ESLint rule https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-default-export.md. I will enable it in the final PR

File: packages/fluentui/react-component-nesting-registry/src/index.ts [hunk 2]
```diff
@@ -1,2 +1,2 @@
-export { default as Unstable_NestingAuto } from './NestingAuto';
```
Comment (miroslavstastny) [inline packages/fluentui/react-component-nesting-registry/src/index.ts:1]: default exports in `react-component-nesting-registry`:
image

Comment (layershifter) [reply packages/fluentui/react-component-nesting-registry/src/index.ts:1]: Fixed 👍

**Pull request merged**